check/asv_run

```diff
@@ -11,5 +11,4 @@
 cd "$(dirname "${BASH_SOURCE[0]}")"
 cd "$(git rev-parse --show-toplevel)"
 
-export ASV_PYTHONPATH="$(git rev-parse --show-toplevel)/examples"
-asv run "$@"
+asv run "$@"
```

Comment:
Nit - text files should end with a newline character.

Reply:
done